config/crd/bases/services.cloud.sap.com_servicebindings.yaml

```diff
@@ -125,6 +125,14 @@ spec:
                   and additional info under single key. Convenient way to store whole
                   binding data in single file when using `volumeMounts`.
                 type: string
+              secretTemplate:
```

Comment:
Should this also document what values and functions the template code can access?

Reply:
done

Reply:
What I meant was documenting https://github.com/SAP/sap-btp-service-operator/pull/398/files#diff-f068acd0279a72df4ca9c7b273ffe5a133f80c60af108ab29e2a33c9e49db4d6R699-R705, i.e. the data that gets passed to the go template.

Reply:
it will be documented in the readme file